### ISSUE TITLE
[Error] Removing rootful container in OpenSUZE Aeon RC2

### ISSUE THREAD

**Issue by Smollejan** [89luca89/distrobox#1400]
**Describe the bug**

distrobox rm --root libvirtd
Removing exported binaries...
Removing container...
/usr/bin/distrobox-rm: line 400: sudo podman: command not found

**To Reproduce**
distrobox rm --root ''id''

**Expected behavior**
Rootful container removed.

**Logs**
[verbose.txt](https://github.com/89luca89/distrobox/files/15370540/verbose.txt)
[podman logs.txt](https://github.com/89luca89/distrobox/files/15370541/podman.logs.txt)

**Desktop (please complete the following information):**
 - Are you using podman, docker or lilipod? 
 Podman
 - Which version or podman, docker or lilipod?
 Podman: 5.0.3
 - Which version of distrobox?
 1.7.2.1
 - Which host distribution? OpenSUSE Aeon
  RC2
 - How did you install distrobox? 
 Pre-installed

**Comment by madness742** [comment]
Take note  of your container id in `sudo podman ps --all` and use `sudo  podman rm [CONTAINER ID]` to remove it.

**Comment by kastl-ars** [comment]
Should this fix already be included in a release? Still hitting this issue with `distrobox-1.7.2.1-1.1.noarch` on openSUSE Kalpa / Tumbleweed.

**Comment by michel-slm** [comment]
Yeah, this landed after 1.7.2.1 - can a new release be cut? Otherwise distros will need to backport this

**Comment by kastl-ars** [comment]
Any news on a new release containing the fix?

**Comment by pythoninthegrass** [comment]
Same issue on distrobox 1.7.2.1. Host OS is Fedora Linux 40.20240827.0 (Budgie Atomic).

Was able to work around it with @madness742's [suggestion](https://github.com/89luca89/distrobox/issues/1400#issuecomment-2131657251):

> Take note of your container id in `sudo podman ps --all` and use `sudo podman rm [CONTAINER ID]` to remove it.
